ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -51,12 +51,10 @@ public struct SignalProducer<T, E: ErrorType> {
 	/// then complete, or immediately send an error, depending on the given
 	/// Result.
 	public init(result: Result<T, E>) {
-		switch result {
```

Comment:
Alright, [filed an issue on `Result` for this](https://github.com/antitypical/Result/issues/36). May have a chance to look at this today.